doc/api/http2.md

```diff
@@ -2918,6 +2918,13 @@ the server, then [`Http2Stream`]()s are destroyed when they time out. If a
 handler is assigned to the request, the response, or the server's `'timeout'`
 events, timed out sockets must be handled explicitly.
 
+### response.aborted
```

Comment:
1. This should go before `#### response.addTrailers(headers)` ABC-wise.
2. Heading level should be increased from `###` to `####`.